aip-console-jenkins/src/main/java/io/jenkins/plugins/aipconsole/FastScanAction.java

```diff
@@ -9,8 +9,8 @@
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
-public class OnboardApplicationFastScanAction extends BaseAction implements RunAction2 {
-    private static final Logger LOGGER = LoggerFactory.getLogger(OnboardApplicationFastScanAction.class);
+public class FastScanAction extends BaseAction implements RunAction2 {
```

Comment:
Renaming action cases jobs based on PIPE failed.
So keep the same name for Jenkins and Give an alias if you want for CLI

Reply:
Changes done